tools/report.sh

```diff
@@ -287,35 +284,9 @@ if [[ -n $EVENTS ]]; then
   echo "$EVENTS" > "$OUT_DIR"/reports/events/events.txt
 fi
 
-get_pod_logs() {
-  local pod="$1"
-  local con="${2-}"
-  if [[ -n $pod ]]; then
-    local names && names=$($KUBE_CLIENT -n "$NAMESPACE" get po "$pod" -o jsonpath='{.spec.containers[*].name}' --ignore-not-found)
-    local count && count=$(echo "$names" | wc -w)
-    local logs
-    if [[ "$count" -eq 1 ]]; then
-      logs="$($KUBE_CLIENT -n "$NAMESPACE" logs "$pod" ||true)"
-      if [[ -n $logs ]]; then printf "%s" "$logs" > "$OUT_DIR"/reports/podlogs/"$pod".log; fi
-      logs="$($KUBE_CLIENT -n "$NAMESPACE" logs "$pod" -p 2>/dev/null ||true)"
-      if [[ -n $logs ]]; then printf "%s" "$logs" > "$OUT_DIR"/reports/podlogs/"$pod".log.0; fi
-    fi
-    if [[ "$count" -gt 1 && -n "$con" && "$names" == *"$con"* ]]; then
-      logs="$($KUBE_CLIENT -n "$NAMESPACE" logs "$pod" -c "$con" ||true)"
-      if [[ -n $logs ]]; then printf "%s" "$logs" > "$OUT_DIR"/reports/podlogs/"$pod"-"$con".log; fi
-      logs="$($KUBE_CLIENT -n "$NAMESPACE" logs "$pod" -p -c "$con" 2>/dev/null ||true)"
-      if [[ -n $logs ]]; then printf "%s" "$logs" > "$OUT_DIR"/reports/podlogs/"$pod"-"$con".log.0; fi
-    fi
-  fi
-}
-
 echo "podlogs"
-mkdir -p "$OUT_DIR"/reports/podlogs
-mkdir -p "$OUT_DIR"/reports/configs
 PODS=$($KUBE_CLIENT get po -l strimzi.io/cluster="$CLUSTER" -o name -n "$NAMESPACE" | cut -d "/" -f 2)
-PODS="$PODS $($KUBE_CLIENT get po -l strimzi.io/cluster="$BRIDGE" -o name -n "$NAMESPACE" | cut -d "/" -f 2)"
-PODS="$PODS $($KUBE_CLIENT get po -l strimzi.io/cluster="$CONNECT" -o name -n "$NAMESPACE" | cut -d "/" -f 2)"
-PODS="$PODS $($KUBE_CLIENT get po -l strimzi.io/cluster="$MM2" -o name -n "$NAMESPACE" | cut -d "/" -f 2)"
+PODS="$PODS $($KUBE_CLIENT get po -l strimzi.io/cluster="$COMP_NAME" -o name -n "$NAMESPACE" | cut -d "/" -f 2)"
```

Comment:
This expects only one comopenent name and that does not seem to make much sense to me. As this is currently centered around the Kafka cluster (which does not make sense, but that is another story), it is natural that you might want to collect the Kafka cluster together with Bridge, Connect and MM2. Not just one of them.

Reply:
Right, going to revert this change. Thanks.